cmd/gateway-gcs.go

```diff
@@ -31,6 +31,8 @@ import (
 	"strings"
 
 	"cloud.google.com/go/storage"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/cloudresourcemanager/v1"
```

Comment:
It won't ideally show.. @vadmeste you might have to do `make pkg-list` to see what is missing.